config/bootstrap.php

```diff
@@ -93,9 +93,9 @@
  * for a very very long time, as we don't want
  * to refresh the cache while users are doing requests.
  */
-if (!Configure::read('debug')) {
-    Configure::write('Cache._cake_model_.duration', '+1 years');
-    Configure::write('Cache._cake_core_.duration', '+1 years');
+if (Configure::read('debug')) {
+    Configure::write('Cache._cake_model_.duration', '+2 minutes');
+    Configure::write('Cache._cake_core_.duration', '+2 minutes');
```

Comment:
If "endless caching" is not an option, then I'd opt for 1 month, at that frame you can easily detect if there are issues alongside re-caching and are not suddenly hit by it "one year after deployment".

For debug caching, what would you suggest if 10 seconds ain't possible?